specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2023-04-01-preview/costmanagement.generatedetailedcostreport.json

```diff
@@ -286,6 +286,14 @@
           "description": "The status of the long running operation.",
           "$ref": "#/definitions/Status"
         },
+        "startTime": {
+          "description": "The startTime of the operation.",
```

Comment:
can you specify the format for date-time?

Reply:
This is exactly exists in generatedetailedcostreport.json for Microsoft.CostManagement stable version 2022-10-01. I needed to make this update to mitigate a breaking change error in the shared branch validation.